openapi3.yaml

```diff
@@ -52,7 +53,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error'
-  /best:
+  /layer/best:
```

Comment:
we said we will remove the term "best" from our code, no?

Reply:
fixed in ee20559